torchaudio/datasets/yesno.py

```diff
@@ -44,6 +44,11 @@ def __init__(
 
         self._parse_filesystem(root, url, folder_in_archive, download)
 
+
+@_mod_utils.deprecated(
+    "donwload_url_to_file will be dericated in next version that is v0.12"
+    "version=0.12"
+)
```

Comment:
This decorator should be added on the implementation of `download_url_to_file`.
Also, `version` has to be keyword argument, not string.